docs/Doxyfile

```diff
@@ -104,71 +104,17 @@ WARN_LOGFILE           =
 # Configuration options related to the input files
 #---------------------------------------------------------------------------
 INPUT                  = \
-\
-    ../src/ripple/app/misc/TxQ.h \
-    ../src/ripple/app/tx/apply.h \
-    ../src/ripple/app/tx/applySteps.h \
-    ../src/ripple/protocol/STObject.h \
-    ../src/ripple/protocol/JsonFields.h \
-    ../src/test/jtx/AbstractClient.h \
-    ../src/test/jtx/JSONRPCClient.h \
-    ../src/test/jtx/WSClient.h \
-    ../src/ripple/consensus/Consensus.h \
-    ../src/ripple/consensus/ConsensusProposal.h \
-    ../src/ripple/consensus/ConsensusTypes.h \
-    ../src/ripple/consensus/DisputedTx.h \
-    ../src/ripple/consensus/LedgerTiming.h \
-    ../src/ripple/consensus/LedgerTrie.h \
-    ../src/ripple/consensus/Validations.h \
-    ../src/ripple/consensus/ConsensusParms.h \
-    ../src/ripple/app/consensus/RCLCxTx.h \
-    ../src/ripple/app/consensus/RCLCxLedger.h \
-    ../src/ripple/app/consensus/RCLConsensus.h \
-    ../src/ripple/app/consensus/RCLCxPeerPos.h \
-    ../src/ripple/app/tx/apply.h \
-    ../src/ripple/app/tx/applySteps.h \
-    ../src/ripple/app/tx/impl/InvariantCheck.h \
-    ../src/ripple/app/consensus/RCLValidations.h \
-    ../src/README.md \
-    ../src/ripple/README.md \
-    ../README.md \
-    ../RELEASENOTES.md \
-    ../docs/CodingStyle.md \
-    ../docs/CheatSheet.md \
-    ../docs/README.md \
-    ../docs/sample_chart.doc \
-    ../docs/HeapProfiling.md \
-    ../docs/Docker.md \
-    ../docs/consensus.md \
-    ../Builds/macos/README.md \
-    ../Builds/linux/README.md \
-    ../Builds/VisualStudio2017/README.md \
-    ../src/ripple/consensus/README.md \
-    ../src/ripple/app/consensus/README.md \
-    ../src/test/csf/README.md \
-    ../src/ripple/basics/README.md \
-    ../src/ripple/crypto/README.md \
-    ../src/ripple/peerfinder/README.md \
-    ../src/ripple/app/misc/README.md \
-    ../src/ripple/app/misc/FeeEscalation.md \
-    ../src/ripple/app/ledger/README.md \
-    ../src/ripple/app/paths/README.md \
-    ../src/ripple/app/tx/README.md \
-    ../src/ripple/proto/README.md \
-    ../src/ripple/shamap/README.md \
-    ../src/ripple/protocol/README.md \
-    ../src/ripple/json/README.md \
-    ../src/ripple/json/TODO.md \
-    ../src/ripple/resource/README.md \
-    ../src/ripple/rpc/README.md \
```

Comment:
Are these READMEs still included in the documentation? I can't seem to find them at https://thejohnfreeman.github.io/rippled/. For instance, where do I find src/ripple/rpc/README.md?

Reply:
They're all under Related Pages: https://thejohnfreeman.github.io/rippled/pages.html

To find a given document, you need to know its title. I'd like to go through and re-title them all to make them easier to find in the Related Pages list. `src/ripple/rpc/README.md` is at https://thejohnfreeman.github.io/rippled/md_ripple_rpc_README.html